### ISSUE TITLE
don't work with absolute path to vault pass in variable "vault_password_file" of ansible.cfg

### ISSUE THREAD

**Issue by fessmage** [codeflows/vaulty#22]
**Describe the bug**
Don't work with absolute path to vault pass in variable "vault_password_file" of ansible.cfg.

/home/user/project/ansible.cfg:
```
[defaults]
vault_password_file=/home/user/secret
```

encrypted vault file located in /home/user/project/vault.yml and vault pass in /home/user/secret

**VS Code and plugin environment**
 - OS: Windows 10 2004 + WSL2 + Ubuntu 20.04
 - VS Code version: 1.45.1
 - Vaulty version: 0.3.0

**Vaulty log**

```
Attempting to delegate to vaulty:/home/user/project/vault.yml#4
Found 1 ansible.cfg file(s) in workspace:
- /home/user/project/ansible.cfg
1 of these are located in parent directories of the Vault file:
- /home/user/project/ansible.cfg
No Ansible configuration found in home directory
Found vault_password_file in /home/user/project/ansible.cfg, resolved /home/user/secret to /home/user/project/home/user/secret
Decrypting vault with arguments "decrypt --vault-password-file=/home/user/project/home/user/secret --output=- /home/user/project/vault.yml"
```

**Comment by codeflows** [comment]
Thanks for the bug report! Fixed and released in 0.3.1 which should be available shortly. If the fix doesn't work for you, just re-open the issue please.

**Comment by fessmage** [comment]
It works! Thanks for fast response.
